feat/docsify

jboix opened the pull request:
## Description

Makes the guidelines easily accessible through a user-friendly static docsify page. See [https://docsify.js.org/](https://docsify.js.org/). See #10 for an overview of the deployment process.

Check out the following screenshots of the new page:

![Screenshot 1](https://github.com/SRGSSR/guilde-plateformes-propres/assets/11271371/a0f2bf7a-77b1-4702-8542-c5ea69f81e98)

![Screenshot 2](https://github.com/SRGSSR/guilde-plateformes-propres/assets/11271371/7f93f30b-6a4c-4964-bb02-965aac1a1e44)

## Changes Made

- The guides have been moved from `docs/guides` to `public/guides`.
- The README now directs users to the URL where the site will be deployed, ensuring a smoother navigation experience.
- The README under `public` folder acts as the index for the docsify page.

jboix committed:
feat: publish a docsify github page

Makes the guidelines easily accessible through a user-friendly static docsify page.
See https://docsify.js.org/

Changes Made:

- The guides have been moved from `docs/guides` to `public/guides`.
- The README now directs users to the URL where the site will be deployed, ensuring a smoother
  navigation experience.
- The README under `public` folder acts as the index for the docsify page.

MGaetan89 commented on docs/README.md:
Instead of duplicating this file in `public`, can't we update simplify it to say something `Please visit [guildes-plateformes-propres](https://srgssr.github.io/guildes-plateformes-propres) to access the documentation`.

defagos replied:
... or maybe to configure things so that files are served from the `docs` directory so that GitHub conventions are preserved?

jboix replied:
I've opted to make the `README` under `docs` a guide to this project itself, i.e.: How do you add a guide and how the site is created. The introduction that was written by @defagos is now only written in the docsify homepage.

jboix committed:
docs: update the readme

Updates the README to serve as a comprehensive guide to the project itself, eliminating redundancy
with the information on the docsify homepage.

The contribution section has been expanded, the project structure is now explained more clearly.
Links have been added to facilitate navigation to important sections and files.
